sql/moz-fx-data-shared-prod/braze_derived/newsletters_v1/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+friendly_name: Braze Newsletters
+description: |-
+  Newsletter Subscription Data from CTMS and Acoustic.
+  
+  See https://mozilla-hub.atlassian.net/browse/DENG-3008
+owners:
+- cbeck@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+  owner: cbeck
+bigquery:
+  time_partitioning:
+    type: day
+    field: last_modified_date
+    require_partition_filter: true
+scheduling:
+  dag_name: bqetl_braze
```

Comment:
so this is now partitioned by time? I thought it needs to build everything?

either way I think this metadata file is a mix of both approaches. For a partitioned table we would need to give the scheduling section a date_partition_parameter ... it defaults to `submission_date` but since you partition on last_modified_date this might be not what we want ... I will have to look into it

Reply:
ah never mind I see that you use the @submission_date i the query so defaulting to it might work